extensions/wikia/VideoHandlers/videoInfo/VideoInfo.class.php

```diff
@@ -253,7 +253,15 @@ public static function newFromTitle( $videoTitle ) {
 
 			$row = $db->selectRow(
 				'video_info',
-				'*',
+				[
+					"video_title",
+					"video_id",
+					"provider",
+					"added_at",
+					"added_by",
+					"duration",
+					"removed",
+				],
```

Comment:
List taken from `self::newFromRow` method below.

Reply:
Wouldn't `self::$fields` suffice?

Reply:
Nope, they have capitalised letters instead of underscores.